### PR TITLE
[FX-92] V11.0.7 beta

### PR DESCRIPTION
**Pull request by jcoughlin11**
# Description
Adds training methods and streamlines firmware version validation.

# Requirements
- [x] Tested on Windows.
- [x] Tested on Linux.
- [x] Tested on RPi.
- [x] Connected ActPack and runs scripts.

### CODE REVIEW

**Comment by jcoughlin11** [comment]
#115 addressed in 4231c05

**Comment by Sotilrac** [comment]
@jcoughlin11 any specific tests we should be running, and what should we expect the result to be?

**Comment by jcoughlin11** [comment]
@Sotilrac I've run the demos on linux, but I still need to run them on Windows and the Pi. I also would like to test the bootloader with it.

**Comment by jcoughlin11** [comment]
@Sotilrac @randop81 This has been tested and should be good to go. If you wouldn't mind reviewing it when you get a chance, I'd appreciate it. Thanks!

File: docs/source/_static/logo-d.svg
Comment (Sotilrac) [inline docs/source/_static/logo-d.svg:1]: Could we use this logo instead:
https://dephy.com/images/logo-dephy.png

Luke has been trying to phase out this "D" logo. But if the logo needs to be tiny, the one linked might not work

**Review by Sotilrac — CHANGES_REQUESTED** [review]
I loe the new docs. Great initiative!

File: docs/source/_static/logo-d.svg
Comment (jcoughlin11) [reply docs/source/_static/logo-d.svg:1]: I tried, but it doesn't appear to have worked: https://flexsea.readthedocs.io/en/latest/#

Would it be possible to get an svg version of that image?

Comment (jcoughlin11) [reply docs/source/_static/logo-d.svg:1]: @Sotilrac  Nvm. I changed the logo. Let me know what you think: https://flexsea.readthedocs.io/en/latest/

Comment (jcoughlin11) [reply docs/source/_static/logo-d.svg:1]: The only other thing is that the D logo is being used as the favicon for the docs. Should that be changed, kept, or do we want to just not have a favicon?

**Review by Sotilrac — APPROVED** [review]

**Pull request merged**